valhalla/userrequests/serializers.py

```diff
@@ -440,6 +440,15 @@ def validate(self, data):
                     time_available = time_allocation.std_allocation - time_allocation.std_time_used
                 elif data['observation_type'] == UserRequest.TOO:
                     time_available = time_allocation.too_allocation - time_allocation.too_time_used
+                    # For Rapid Response observations, check if the end time of the window is within
```

Comment:
Excuse me butting in (uninvited) to your indent-fest, but is this logic right? Should we be checking for the existence of windows within 6 hours or for the existence of windows beyond 6 hours?

Reply:
The logic is if we find any window with an end time greater than 6 hours + the observation duration (in case its a long observation), then we reject it. I think this is correct